drivers/gpio/gpio_mchp_xec.c

```diff
@@ -42,23 +47,31 @@ static int gpio_xec_configure(struct device *dev,
 {
 	const struct gpio_xec_config *config = dev->config->config_info;
 	__IO u32_t *current_pcr1;
-	u32_t pcr1 = 0;
-	u32_t mask = 0;
-	u32_t gpio_interrupt = 0;
+	u32_t pcr1 = 0U;
+	u32_t mask = 0U;
+	__IO u32_t *gpio_base = GPIO_OUT_BASE(config);
```

Comment:
The name `gpio_base` is a bit misleading since we get a direct access to GPIO output register rather than GPIO module base. What about changing the name to `gpio_out_reg` or something similar?